rubocop-airbnb/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+#### Summary
+
+<!--- required --->
+
+#### Test Plan: what steps did you take to test this?
+
+<!--- required --->
+
+#### Reviewers
+
+<!--- required --->
+
+#### JIRA Task
+
+<!--- optional --->
```

Comment:
JIRA Task doesn't make sense in open source

Reply:
ah thought i'd removed this.